assets/js/vue/components/search/SearchForm.vue

```diff
@@ -153,14 +153,21 @@
                 this.form.page = newPageNo;
                 this.onSubmit();
             },
+            detectHashChange: function () {
+                this.route = window.location.hash;
+                if (this.route) {
+                    const urlHashSplit = this.route.split("#")[1].split("&").map(value => value.split("="));
+                    this.form = Object.fromEntries(urlHashSplit);
```

Comment:
If there are no entries, it won't work. For instance: start with a clean search, then search for something, then go back. (Basically a blank page) and it won't work.

Reply:
Will fix it, didn't think about this scenario

Reply:
Fixed it